setup.py

```diff
@@ -52,6 +52,7 @@
         'PyGithub',          # nice to have
     ],
     'misc': [
+        'helpme[github]',    # support for helpme error submission
```

Comment:
Let's put current version of helpme as minimally required

Reply:
got it.